x-pack/plugins/security_solution/public/common/components/guided_onboarding_tour/tour_step.tsx

```diff
@@ -89,11 +117,13 @@ export const SecurityTourStep = ({ children, step, stepId }: SecurityTourStep) =
   //  see type EuiTourStepAnchorProps
   return anchor != null ? (
     <>
-      <EuiTourStep {...commonProps} anchor={anchor} />
+      <StyledTourStep stepId={stepId} {...commonProps} anchor={anchor} />
```

Comment:
NIT: In this case where we have the `anchor`, do we need to render the TourStep if we don't have `children`? 
I am asking because if we don't have to render anything if `children` is not passed, then we could create an exit case (`if !children return null`) much earlier in the component, and avoid checking it on every return, and this double ternary condition as well.

Reply:
Yes, step 5 is only the tour step, no children passed